pkg/receive/multitsdb.go

```diff
@@ -265,7 +262,7 @@ func (t *MultiTSDB) TSDBStores() map[string]storepb.StoreServer {
 
 func (t *MultiTSDB) startTSDB(logger log.Logger, tenantID string, tenant *tenant) error {
 	reg := prometheus.WrapRegistererWith(prometheus.Labels{"tenant": tenantID}, t.reg)
-	lset := labelpb.ExtendSortedLabels(t.labels, labels.FromStrings(t.tenantLabelName, tenantID))
+	lbls := append(t.labels, labels.Label{Name: t.tenantLabelName, Value: tenantID})
```

Comment:
I guess we still want to have `labelpb.ExtendSortedLabels` here

Reply:
Okay, reverting back